crane/crane.go

```diff
@@ -92,6 +94,19 @@ func checkDockerClient() {
 			printErrorf("Unsupported client version! Please upgrade to Docker %v or later.\n", intJoin(requiredDockerVersion, "."))
 		}
 	}
+	actualDockerVersion = versions
+}
+
+func validateDockerClientAbove(expectedVersions []int) bool {
```

Comment:
IMHO, Docker 1.6 is a good base version. I think enabling features based on their API availability is better for users with multiple docker engine version (which, in large deployments, is a reasonable scenario)